setup/src/Magento/Setup/Test/Unit/Module/Di/Code/Reader/ClassesScannerTest.php

```diff
@@ -12,9 +12,18 @@ class ClassesScannerTest extends \PHPUnit_Framework_TestCase
      */
     private $model;
 
+    /**
+     * the /var/generation directory realpath
+     *
+     * @var string
+     */
+
+    private $generation;
+
     protected function setUp()
     {
-        $this->model = new \Magento\Setup\Module\Di\Code\Reader\ClassesScanner();
+        $this->generation = realpath(__DIR__ . '/../../_files/var/generation');
```

Comment:
To get a clear indication that it is the path that is broken if there is an error during debugging.